scripts/grok1_multiblock_experiment.py

```
@@ -740,9 +896,10 @@ def _remedy_issue_meta(*, v2: bool, v3: bool, v4: bool = False) -> tuple[str, st
         return (
             "GH #85 / Linear RM-608 / beads goz-3h3",
             REMEDY_V4_AGENT_LINE,
-            "Grok-4.5",
-            "Grok Build design lock: INT4 codes × LS channel-α stack at Grok-1 "
-            "max context (8192); re-measure INT4 baseline same-budget; #80@2048 historical only",
+            "OpenAI Codex",
+            "Grok issue-planning lock: INT4 codes × LS channel-α stack at Grok-1 "
+            "max context (8192); re-measure INT4 baseline same-budget; #80@2048 "
+            "historical only; implementation and evidence by Codex (OpenAI)",
```

Ruff (0.16.2)

[warning] Unparenthesized implicit string concatenation in collection

Did you forget a comma?

(ISC004)